portfolio/src/main/webapp/index.html

```diff
@@ -121,10 +121,13 @@
             </tr>
         </table>
     </div>
-    <div class="comments">
-        <textarea id="comment-box"></textarea>
-    </div>
-    <div id="display-comments">
-    </div>
+    <form action="/data" method="POST">
+        <div class="comments">
+            <textarea name="comment-box"></textarea>
+            <input type="submit"/>
+        </div>
+        <div id="display-comments">
```

Comment:
I don't think you want this div inside your form since it doesn't interact with it. This way it's clearer to readers what is in the form and what is not.

Reply:
i plan to use this class soon for styling the comments